src/CMakeLists.txt

```diff
@@ -62,13 +62,7 @@ configure_file(
   
 
 ############ nupic_core Library ######################################
-# Setup nupic_core static library, consisting of our own sources and yaml libraries;
-#
-# this is an intermediate archive that will be merged with external static
-# libraries in a subsequent step to produce the nupic_core "combined" static
-# library.
-#
-set(src_lib_static_nupiccore_solo nupiccore_solo)
```

Comment:
@dkeeney I remembered there was a reason we chose static. Couldn’t all external and ours be made shared?

Reply:
There are times when building a C++ app where you really want a static lib.  I think we need to provide both.